tox.ini

```diff
@@ -16,11 +16,13 @@ tox_pyenv_fallback = false
 
 [testenv]
 skip_install = true
+setenv =
+    dev: DEBUG = {env:DEBUG:yes}
+    dev: HYPOTHESIS_AUTHORITY = {env:HYPOTHESIS_AUTHORITY:localhost}
+    dev: HYPOTHESIS_URL = {env:HYPOTHESIS_URL:http://localhost:5000}
+    dev: VIA_BASE_URL = {env:VIA_BASE_URL:http://localhost:9080}
 passenv =
     dev: CHROME_EXTENSION_ID
-    dev: DEBUG
-    dev: HYPOTHESIS_AUTHORITY
-    dev: HYPOTHESIS_URL
```

Comment:
These are no longer needed in passenv since the env subsitutions in setenv now read them